website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -39,17 +39,19 @@ Consul supports **four major Envoy releases** at the beginning of each major Con
 
 | Consul Version      | Compatible Envoy Versions                                                          |
 | ------------------- | -----------------------------------------------------------------------------------|
-| 1.18.x              | 1.28.0, 1.27.2, 1.26.6, 1.25.11                                                    |
-| 1.17.x              | 1.27.2, 1.26.6, 1.25.11, 1.24.12                                                   |
-| 1.16.x              | 1.26.6, 1.25.11, 1.24.12, 1.23.12                                                  |
+| 1.18.x              | 1.28.1, 1.27.3, 1.26.7, 1.25.11                                                    |
+| 1.17.x              | 1.27.3, 1.26.7, 1.25.11, 1.24.12                                                   |
+| 1.16.x              | 1.26.7, 1.25.11, 1.24.12, 1.23.12                                                  |
+| 1.15.x              | 1.28.1, 1.27.3, 1.26.7, 1.25.11, 1.26.7, 1.25.11, 1.24.12, 1.23.12                 |
 
 ### Envoy and Consul Dataplane
 
 The Consul dataplane component was introduced in Consul v1.14 as a way to manage Envoy proxies without the use of Consul clients. Each new minor version of Consul is released with a new minor version of Consul dataplane, which packages both Envoy and the `consul-dataplane` binary in a single container image. For backwards compatibility reasons, each new minor version of Consul will also support the previous minor version of Consul dataplane to allow for seamless upgrades. In addition, each minor version of Consul will support the next minor version of Consul dataplane to allow for extended dataplane support via newer versions of Envoy.
 
 | Consul Version      | Default `consul-dataplane` Version                          | Other compatible `consul-dataplane` Versions |
 | ------------------- | ------------------------------------------------------------|----------------------------------------------|
-| 1.17.x              | 1.3.x (Envoy 1.27.x)                                        | 1.2.x (Envoy 1.26.x)                         |
+| 1.18.x              | 1.4.x (Envoy 1.28.x)                                        | 1.3.x (Envoy 1.27.x)                         |
+| 1.17.x              | 1.3.x (Envoy 1.27.x)                                        | 1.4.x (Envoy 1.28.x), 1.2.x (Envoy 1.26.x)   |
```

Comment:
@david-yu could I get another look at this change to 👍🏻 the docs here? I noticed they were a bit inconsistent.

I know we sometimes have updated these closer to the actual .0 release, but given we already had an entry for 1.18.x in one table, probably ok to just keep between now and the release (given we're also already publishing dev images, and past RC1)

Reply:
This looks good to add these in here. I'm curious about the number of Envoy versions that 1.15.x supports. Will take offline.

Reply:
Cool, thanks - for better visibility for others, this is essentially a follow-up to https://github.com/hashicorp/consul/commit/9bf543503c9b06add7b4cd223449892f482f3eca